use-timescale/vpc.md

```diff
@@ -69,9 +77,21 @@ between $CLOUD_LONG and your own VPC in a logically isolated virtual network.
 
 1.  In [$CONSOLE > VPC][console-vpc], click `New VPC`.
 
-    The number of VPCs you can attach to your project depends on your [pricing plan][pricing-plans]. If you
-    need more VPCs either contact contact [support@timescale.com](mailto:support@timescale.com) or change 
-    your pricing plan in $CONSOLE. Each $CLOUD_LONG VPC can have as many peering connections as you need.
+    ![$CLOUD_LONG new $VPC](https://assets.timescale.com/docs/images/console-add-vpc.png)
```

Comment:
Don't think variables will work in the alt tag

Reply:
They do :-).